packages/access-client/src/agent.js

```diff
@@ -141,7 +181,6 @@ export class Agent {
    */
   async addProof(delegation) {
     validate(delegation, {
-      checkAudience: this.issuer,
```

Comment:
I don't think removing this check should be necessary - added it back and tested the whole flow manually and it worked. I tried to add it back but there's a an access-api test that's failing, @gobengo is planning on looking into why there's a discrepancy between the test and the observed behavior

Reply:
together we realized @travis is right. It was only necessary because my test was explicitly trying to claim delegations with=account, which I removed.

Reply:
https://github.com/web3-storage/w3protocol/pull/433/commits/70b663d534edf96959d86fd98f372fcbfc2c4fdc